### PR TITLE
Fix MultiSelect group display when filtering

### PR DESCRIPTION
**Pull request by john-oc**
Issue created:- https://github.com/primefaces/primevue/issues/3068

### CODE REVIEW

**Comment by tugcekucukoglu** [comment]
Thanks for the PR. We want to merge it but this PR needs a format check. Please create `settings.json` and then copy-paste the [https://github.com/primefaces/primevue/blob/master/.vscode/settings.team.json](https://github.com/primefaces/primevue/blob/master/.vscode/settings.team.json) in your local and run the `format:check` command.

**Comment by john-oc** [comment]
Formatting updated as requested.

File: src/components/multiselect/MultiSelect.vue [hunk 1]
```diff
@@ -1063,7 +1063,23 @@ export default {
...
+            const filteredOptions = FilterService.filter(options, this.searchFields, this.filterValue, this.filterMatchMode, this.filterLocale);
+
+            if (!this.optionGroupLabel) return filteredOptions;
+
+            const optionGroups = this.options || [];
+
+            const filtered = [];
+
+            optionGroups.forEach((group) => {
+                const filteredItems = group.items.filter((x) => filteredOptions.includes(x));
```
Comment (bahadirsofuoglu) [inline src/components/multiselect/MultiSelect.vue:1077]:
```suggestion
                const filteredItems = group.items.filter((item) => filteredOptions.includes(item));
```
Simple change. Thank you for your contribution.

Comment (john-oc) [reply src/components/multiselect/MultiSelect.vue]: Updated variable name as suggested

**Pull request merged**